src/foam/u2/dialog/ApplicationPopup.js

```diff
@@ -62,6 +63,8 @@ foam.CLASS({
 
     ^body {
       flex-grow: 1;
+      max-height: 90vh;
+      overflow: scroll;
```

Comment:
Should be `overflow: auto`

Reply:
https://github.com/kgrgreer/foam3/pull/1806